qa/L0_model_config/test.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright (c) 2019-2021, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2019-2022, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Should be `Copyright 2019-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
`, otherwise the change looks good to me